<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pamietnik</class>
 <widget class="QWidget" name="Pamietnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>752</width>
    <height>550</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Pamietnik</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lista wpisów</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit_3"/>
      <widget class="QTextEdit" name="textEdit_2"/>
      <widget class="QTextEdit" name="textEdit_4"/>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Dark mode</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="dodajButton">
      <property name="text">
       <string>Dodaj wpis</string>
      </property>
     </widget>
     <widget class="QPushButton" name="usunButton">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Następny</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prevButton">
      <property name="text">
       <string>Poprzedni</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="button_StworzKopie">
       <property name="text">
        <string>Kopia zapasowa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_PrzywrocKopie">
       <property name="text">
        <string>Przywróć kopię</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Filtry</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="smutneBox">
           <property name="text">
            <string>Smutne</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="NeutralneBox">
           <property name="text">
            <string>Neutralne</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="RadosneBox">
           <property name="text">
            <string>Radosne</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filtrButton">
           <property name="text">
            <string>Filtruj</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
